project/ScoverageSettings.scala

```diff
@@ -1,4 +1,4 @@
-import scoverage.ScoverageKeys._
+import scoverage.ScoverageKeys.*
```

Comment:
Scala 3 syntax already ?  Bit keen

Reply:
It seems to work and it removes a warning from IntelliJ. I can revert if you prefer but I've done it elsewhere too.

Reply:
I've set it back to `_`

Reply:
IntelliJ is getting ahead of itself... probably thinks we should be using Scala 3 after it's been out for 5 years :-)